src/waveform/renderers/waveformrendermark.cpp

```diff
@@ -202,7 +202,7 @@ void WaveformRenderMark::generateMarkImage(WaveformMark& mark) {
         QColor lineColor = mark.m_color;
         lineColor.setAlpha(140);
         painter.setPen(lineColor);
-        float middle = mark.m_image.width()/2;
+        float middle = mark.m_image.width()/2.0;
```

Comment:
No need for a change since it is not you change, but we use space surrounded operators. It would be nice to fix this in future changes. Thanks.